scripts/sanitise-db.sql

```diff
@@ -339,9 +361,17 @@ DO $$
 BEGIN
     PERFORM anonymize_with_lorem_ipsum('funding', 'data_blob', 'funding_source');   --- LARGE TABLE
     PERFORM anonymize_with_lorem_ipsum('funding_comment', 'data_blob', 'comment');
+    PERFORM anonymize_with_lorem_ipsum('funding_question', 'data_blob', 'question');
```

Comment:
Could you confirm why these specific fields are being sanitised and not other fields in the `funding_question` data blob including:

- response
- guidance_notes

Reply:
Same like above :)

Reply:
Same as above for me too 😄 although quick note - I would have thought the "response" field here could potentially contain some stuff that we'd want to sanitise even if we weren't being super-cautious